content/docs/start/experiments.md

```diff
@@ -226,55 +305,48 @@ Let's take another look at the experiments table:
 experiments from the previous _n_ commits:
 
 ```dvc
 $ dvc exp show -n 2 --no-timestamp \
-                    --include-params train.n_est,train.min_split
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
-│ workspace     │  0.60405 │  0.9608 │ 100        │ 64              │
-│ master        │  0.60405 │  0.9608 │ 100        │ 64              │
-│ 64d74b2       │  0.55259 │ 0.91536 │ 50         │ 2               │
-│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
-│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
-│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
-│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
-│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
-│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
-└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+               --include-params model.cnn.conv_units \
+               --include-metrics categorical_accuracy
 ```
 
+![dvc exp show screenshot](/img/doc/start/exp-ss-68591.png)
+
 Eventually, old experiments may clutter the experiments table.
 
 `dvc exp gc` removes all references to old experiments:
 
 ```dvc
 $ dvc exp gc --workspace
+...
+Removed 5 experiments. To remove unused cache files use 'dvc gc'.
 $ dvc exp show -n 2 --no-timestamp \
-                    --include-params train.n_est,train.min_split
-┏━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
-┡━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
-│ workspace  │  0.60405 │  0.9608 │ 100        │ 64              │
-│ master     │  0.60405 │  0.9608 │ 100        │ 64              │
-│ 64d74b2    │  0.55259 │ 0.91536 │ 50         │ 2               │
-└────────────┴──────────┴─────────┴────────────┴─────────────────┘
+               --include-params model.cnn.conv_units \
+               --include-metrics categorical_accuracy
 ```
 
-> `dvc exp gc` only removes references to the experiments; not the cached
-> objects associated with them. To clean up the <abbr>cache</abbr>, use
-> `dvc gc`.
+![dvc exp show screenshot](/img/doc/start/exp-ss-32408.png)
+
+> `dvc exp gc` only removes references to the experiments, not the cached
+> objects associated to them. To clean up the cache, use `dvc gc`.
+
+## Going Further
+
+You can continue to experiment with
+[the project](https://github.com/iterative/get-started-experiments). There are
+many parameters available to modify, e.g., `noise` or `units` in Dense layer of
+the network. Please see the `README.md` file of the project for these. Don't
+forget to [notify us](https://dvc.org/chat) if you happen to find good
+parameters.
```

Comment:
Is this to encourage community engagement?

Reply:
Just curious.

Reply:
Yes. I would like the users to play with the repo and come up with higher metrics just using the hyperparameters. We can have a _hall of fame_ somewhere maybe.

Reply:
Good idea @iesahin !!

Reply:
I removed this part in #2574 because the repo is very simple now. Maybe we can have multiple versions in the future.